.github/workflows/ci.yml

```
@@ -11,21 +11,14 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, macOS-latest, windows-latest]
-        cabal: ["3.4"]
+        os:
+          - ubuntu-latest
+        cabal:
+          - 3.4
```

I'd change this to 3.6, just to be on the latest recommended version
```suggestion
          - 3.6
```